src/views/bridge-details/bridge-details.view.tsx

```diff
@@ -267,9 +271,11 @@ const BridgeDetails: FC = () => {
       return <Navigate to={routes.activity.path} replace />;
     }
     case "successful": {
-      const { status, amount, from, to, token, depositTxHash } = bridge.data;
+      const { status, amount, from, to, token } = bridge.data;
 
-      const bridgeTxUrl = `${from.explorerUrl}/tx/${depositTxHash}`;
+      const bridgeTxUrl = `${from.explorerUrl}/tx/${
+        bridge.data.status === "pending" ? bridge.data.depositTxHash : bridge.data.depositTxHash
```

Comment:
:neutral_face:

Reply:
This view will be improved in next iterations. It has not been taken into account by the moment

Reply:
LOL right! What an amazing ternarny I did 🤣 Fixed!